test/cli/install/bun-install.test.ts

```diff
@@ -4519,6 +4519,42 @@ it("should fail on invalid Git URL", async () => {
   }
 });
 
+it("should fail on ssh Git URL if invalid credentials", async () => {
+  const urls: string[] = [];
+  setHandler(dummyRegistry(urls));
+  await writeFile(
+    join(package_dir, "package.json"),
+    JSON.stringify({
+      name: "Foo",
+      version: "0.0.1",
+      dependencies: {
+        "private-install": "git+ssh://git@bitbucket.org/kaizenmedia/private-install-test.git",
+      },
+    }),
+  );
+  const { stdout, stderr, exited } = spawn({
+    cmd: [bunExe(), "install"],
+    cwd: package_dir,
+    stdout: "pipe",
+    stdin: "pipe",
+    stderr: "pipe",
+    env,
```

Comment:
```suggestion
    stdin: "ignore",
    stderr: "pipe",
    env: { ...env, "GIT_ASKPASS": "echo" },
```
to prevent prompting for a password when running the test